org.kde.AwesomeBump.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.kde.AwesomeBump",
+    "branch": "Release",
+    "build-options": {
+        "no-debuginfo": true,
+        "strip": true
+    },
+    "command": "awesomebump",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=system-bus",
+        "--device=dri",
+        "--filesystem=home",
+        "--own-name=org.kde.AwesomeBump"
+    ],
+    "modules": [
+        {
+            "name": "AwesomeBump",
+            "no-autogen": true,
+            "no-make-install": false,
+            "no-parallel-make": false,
+            "sources": [
+                {                    
+                    "tag": "Linuxv5.1.1",
+                    "commit": "d6912bfcb5219844841c2582f799c9a6dddaf533",
+                    "type": "git",
+                    "url": "https://github.com/kmkolasinski/AwesomeBump"
+                },
+                {
+                    "commands": [
+                       "qmake ./AwesomeBump.pro", "make -j 8", "rm .qmake.stash",
+                       "make clean", "rm Makefile Sources/Makefile", "qmake ./AwesomeBump.pro 'CONFIG+=gl330'", "make -j 8"
+                    ],
+                    "dest-filename": "configure",
+                    "type": "script"
+                }
+            ],
+            "build-commands": [
+                "install -D /run/build/AwesomeBump/workdir/linux-g++-gl4/bin/AwesomeBump /app/bin/AwesomeBump",
+                "install -D /run/build/AwesomeBump/workdir/linux-g++-gl3/bin/AwesomeBump /app/bin/AwesomeBumpGL330",
+                "cp -r /run/build/AwesomeBump/Bin/Configs /app/bin/Configs",
+                "cp -r /run/build/AwesomeBump/Bin/Core /app/bin/Core", 
+                "echo '#!/bin/bash\ncd /app/bin/\n./AwesomeBump' > /app/bin/awesomebump", "chmod +x /app/bin/awesomebump",
+                "echo '#!/bin/bash\ncd /app/bin/\n./AwesomeBumpGL330' > /app/bin/awesomebump-gl330", "chmod +x /app/bin/awesomebump-gl330"
+            ]
+        }
+    ],
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
```

Comment:
fwiw the 5.13 runtime is now available which is based on the Qt 5.13 release and the Freedesktop Platform 19.08, so it will have a longer duration in security support than 5.12. It's your choice though.